docs/source/conf.py

```diff
@@ -38,9 +38,11 @@
 extensions = [
     'sphinx.ext.autodoc',
     'sphinx.ext.viewcode',
+    'sphinx.ext.githubpages',
```

Comment:
Just curious. What does this do? Make the `.nojekyll` file?

Reply:
Yes! Exactly! 👍